[2023-10-26T07:07:54.818+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load_data manual__2023-10-26T07:00:01.517630+00:00 [queued]>
[2023-10-26T07:07:54.847+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load_data manual__2023-10-26T07:00:01.517630+00:00 [queued]>
[2023-10-26T07:07:54.849+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2023-10-26T07:07:54.872+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): load_data> on 2023-10-26 07:00:01.517630+00:00
[2023-10-26T07:07:54.886+0000] {standard_task_runner.py:57} INFO - Started process 109 to run task
[2023-10-26T07:07:54.890+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load_data', 'manual__2023-10-26T07:00:01.517630+00:00', '--job-id', '462', '--raw', '--subdir', 'DAGS_FOLDER/tugaskel8_dag.py', '--cfg-path', '/tmp/tmp8ip62pxc']
[2023-10-26T07:07:54.895+0000] {standard_task_runner.py:85} INFO - Job 462: Subtask load_data
[2023-10-26T07:07:54.979+0000] {task_command.py:415} INFO - Running <TaskInstance: etl_dag.load_data manual__2023-10-26T07:00:01.517630+00:00 [running]> on host d73c1ff20372
[2023-10-26T07:07:55.695+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kel8' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-26T07:00:01.517630+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-26T07:00:01.517630+00:00'
[2023-10-26T07:07:55.775+0000] {base.py:73} INFO - Using connection ID 'db_polusiudara' for task execution.
[2023-10-26T07:07:57.534+0000] {python.py:194} INFO - Done. Returned value was: Sukses Memasukkan Data
[2023-10-26T07:07:57.574+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load_data, execution_date=20231026T070001, start_date=20231026T070754, end_date=20231026T070757
[2023-10-26T07:07:57.637+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-26T07:07:57.669+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
